generate_stats.py

```diff
@@ -32,11 +31,10 @@ def main():
 
     # get tags
     dagmc_tags = dagmc_stats.get_dagmc_tags(my_core)
-    
+    print(dagmc_tags)
```

Comment:
Is this for debugging or part of the normal output that you expect?

Reply:
Debugging, it is also gone in more recent versions